medic/medic-run.js

```diff
@@ -232,12 +233,18 @@ function windowsSpecificPreparation(argv) {
             /^\s*\$appActivator .*$/gim,
             "$&\n" +
```

Comment:
This was copied over from the old Medic code. Having said that, improvements are in order at some point.